wm/frame.go

```diff
@@ -356,6 +359,11 @@ func (f *frame) applyTheme(force bool) {
 	}
 }
 
+func (f *frame) setTitle() {
```

Comment:
A setter should take a parameter - this probably needs a different name?

Reply:
Fixed.